mdm/api/src/main/java/com/gentics/mesh/parameter/impl/SortingParametersImpl.java

```diff
@@ -71,6 +71,8 @@ public String getName() {
 	public SortingParameters putSort(String sortBy, SortOrder order) {
 		if (StringUtils.isNotBlank(sortBy) && order != null) {
 			sort.put(sortBy, order);
+			setParameter(SORT_BY_PARAMETER_KEY, sortBy);
```

Comment:
This fix allows reusing PagingParametersImpl for sorting only, at the server side.